public/app/features/admin/AdminEditOrgPage.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC, useState, useEffect } from 'react';
+import Page from 'app/core/components/Page/Page';
+import { useSelector } from 'react-redux';
+import { StoreState, OrgUser } from 'app/types';
+import { getNavModel } from 'app/core/selectors/navModel';
+import UsersTable from '../users/UsersTable';
+import { useAsyncFn } from 'react-use';
+import { getBackendSrv } from '@grafana/runtime';
+import { UrlQueryValue } from '@grafana/data';
+import { Form, Field, Input, Button, Legend } from '@grafana/ui';
+import { css } from 'emotion';
+
+interface OrgNameDTO {
+  orgName: string;
+}
+
+const getOrg = async (orgId: UrlQueryValue) => {
+  return await getBackendSrv().get('/api/orgs/' + orgId);
+};
+
+const getOrgUsers = async (orgId: UrlQueryValue) => {
+  return await getBackendSrv().get('/api/orgs/' + orgId + '/users');
+};
+
+const updateOrgUserRole = async (orgUser: OrgUser, orgId: UrlQueryValue) => {
+  await getBackendSrv().patch('/api/orgs/' + orgId + '/users/' + orgUser.userId, orgUser);
+};
+
+const removeOrgUser = async (orgUser: OrgUser, orgId: UrlQueryValue) => {
+  return await getBackendSrv().delete('/api/orgs/' + orgId + '/users/' + orgUser.userId);
+};
+
+export const AdminEditOrgPage: FC = () => {
+  const navIndex = useSelector((state: StoreState) => state.navIndex);
+  const navModel = getNavModel(navIndex, 'global-orgs');
+
+  const orgId = useSelector((state: StoreState) => state.location.routeParams.id);
+
+  const [users, setUsers] = useState<OrgUser[]>([]);
+
+  const [orgState, fetchOrg] = useAsyncFn(() => getOrg(orgId), []);
+  const [, fetchOrgUsers] = useAsyncFn(() => getOrgUsers(orgId), []);
```

Comment:
```suggestion
  const [, fetchOrgUsers] = useAsyncFn(() => getOrgUsers(orgId), []);
```

This is interesting never seen this syntax before :)